db/pebble/db.go

```diff
@@ -17,6 +17,7 @@ const (
 	// minCache is the minimum amount of memory in megabytes to allocate to pebble read and write caching.
 	// This is also pebble's default value.
 	minCacheSizeMB = 8
+	byteLimit      = 0xff
```

Comment:
IMO better describes / more  idiomatic - but subjective. Feel free to ignore
```suggestion
	maxByte = ^byte(0)
```